packages/utilities/psammead-locales/moment/yo.js

```diff
@@ -0,0 +1,15 @@
+const moment = require('moment');
+
+moment.updateLocale('yo', {
+  relativeTime: {
+    // these keys have been overridden with our translations
+    // some other keys have been left as set upstream
+    // see https://github.com/moment/moment/blob/develop/src/locale/yo.js
+    past: '%s sẹ́yìn',
+    s: 'ìṣẹ́jú aayá die',
```

Comment:
Does this need to be overridden? Looks the same as https://github.com/moment/moment/blob/develop/src/locale/yo.js#L32

Reply:
ìṣẹ́jú and ìsẹjú are accented differently